backend/tests/routes/test_geo_stats.py

```diff
@@ -0,0 +1,67 @@
+from http.client import NOT_FOUND, OK, UNAUTHORIZED
+from unittest.mock import Mock
+from app.initial_data import populate_initial_data
+
+from app.api.api_v1.routers.lookups.geo_stats import (
+    GeoStatsResponse,
+    lookup_geo_stats,
+)
+
+TEST_ID = 11
+TEST_GEO_NAME = "Antigua and Barbuda"
+URL_UNDER_TEST = f"/api/v1/geo_stats/{TEST_ID}"
+
+
+def test_endpoint_security(client):
+    response = client.get(URL_UNDER_TEST)
+    assert response.status_code == UNAUTHORIZED
+
+
+def test_endpoint_returns_correct_data(client, user_token_headers, test_db):
+    populate_initial_data(test_db)
+    test_db.flush()  # update the session, no need to commit as its just a test
+
+    response = client.get(
+        URL_UNDER_TEST,
+        headers=user_token_headers,
+    )
+    stats = response.json()
+    assert response.status_code == OK
+    assert stats["id"] == 6
+    assert stats["geography_id"] == TEST_ID
+    assert stats["name"] == TEST_GEO_NAME
+
+
+def test_endpoint_returns_not_found(client, user_token_headers, test_db):
+    response = client.get(
+        URL_UNDER_TEST,
+        headers=user_token_headers,
+    )
+    assert response.status_code == NOT_FOUND
```

Comment:
I'm nervous that this test is quite loose - if there's no data in the database are we sure this is actually testing the case when there are no geostats for a given id...?

Reply:
I've commented and added another test to explain a little more

Reply:
:+1: